src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -332,9 +333,30 @@ public AdsBaseObject verifyProvidersChain(String slotName, String providers) {
    * Test whether the correct GPT ad unit is called
    *
    * @param adUnit the ad unit passed to GPT, like wka.wikia/_wikiaglobal//home
+   * @param slotName the name of the slot an ad is going to be inserted into
+   * @param src the source of an ad, for example gpt or remnant
    */
   public void verifyGptIframe(String adUnit, String slotName, String src) {
-    String iframeId = "google_ads_iframe_/5441/" + adUnit + "/" + src + "/" + slotName + "_0";
+    verifyGptIframe(WIKIA_DFP_CLIENT_ID, adUnit, slotName, src);
+  }
+
+  /**
+   * Test whether the correct GPT ad unit is called
+   *
+   * @param dfpClientId in most cases it's Wikia id but we have other partners like Evolve or Turtle
+   * @param adUnit the ad unit passed to GPT, like wka.wikia/_wikiaglobal//home
+   * @param slotName the name of the slot an ad is going to be inserted into
+   * @param src the source of an ad, for example gpt, remnant or empty
+   */
+  public void verifyGptIframe(int dfpClientId, String adUnit, String slotName, String src) {
+    String iframeId;
+
+    if(src.isEmpty()) {
+      iframeId = "google_ads_iframe_/" + dfpClientId + "/" + adUnit + "/" + slotName + "_0";
+    } else {
+      iframeId = "google_ads_iframe_/" + dfpClientId + "/" + adUnit + "/" + src + "/" + slotName + "_0";
+    }
```

Comment:
What about get rid of this `if` and make `verifyGptIframe` even more beautiful? :smirk_cat: 

1. Change `src` param to be optional.   
It will allow to call method without `""`: `ads.verifyGptIframe(dfpClientId, adUnit, slot);`
2. Get rid of `if` and use Guava `Joiner ` instead.

```java
  public void verifyGptIframe(int dfpClientId, String adUnit, String slotName, String... src) {
    String iframeId = Joiner.on("/").skipNulls().join(
            "google_ads_iframe_",
            String.valueOf(dfpClientId),
            adUnit,
            src.length > 0 ? src[0] : null,
            slotName + "_0"
        );
    ...
}
```